css/index.css

```diff
@@ -0,0 +1,174 @@
+    margin-left: 20px;
+    margin-top: 20px;
+}
+
+.timer-container {
+    background-color: #ffffff49;
+    color: rgb(101, 9, 6);
+    text-align: center;
+    margin: auto;
+    width: 20%;
+    padding: 10px;
+    /* justify-content: center; */
+    
+    
+    /* font-size: large;
+    border: 2px solid;
+    margin-left: 40%;
+    margin-right: 40%; */
+    
+}
+
+.time-val {
+    font-size: 50px;
+    /* text-shadow: none; */
+    color: black;
+}
+
+.main {
+    display: grid;
+    grid-template-columns: 20% 60% 20%;
+    gap: 20px;
+    margin: 10px 30px;
+    text-align: center;
+    justify-content: center;
+}
+
+.home-score, .guest-score {
+    background-color: #da454549;
+    justify-content: center;
+    color: white;
+    margin-bottom: auto;
+}
+
+.score-summary {
+    display: flex;
+    flex-direction: column;
+    row-gap: 15px;
+    padding-top: 15px;
+}
+
+.round-score {
+    display: flex;
+    justify-content: center;
+    font-size: 20px;
+    font-weight: bold;
+    background-color: rgba(128, 128, 128, 0.356);
+    color: black;
+    border: 3px solid black;
+    margin: 5px 0px;
+    /* text-align: center; */
+    
+    
+}
+.score {
+    display: grid;
+    grid-template-columns: 90% 10%;
+    border: 10px solid wheat;
+}
+
+.point-board {
+    min-height: 350px;
+    text-align: center;
+    background-color: #ffffff49;
+}
+
+.score-heading {
+    background-image: linear-gradient(rgba(0, 128, 128, 0.738), rgb(0, 229, 255), salmon);
+    /* background-color: rgba(222, 59, 37, 0.87); how to give linear-gradient property for multicolor background? */
+    color: black;
+    text-shadow: 1px 1px rgba(0, 0, 255, 0.434);
+    border-bottom: 4px solid rgb(34, 34, 48);
+}
+.game-name {
+    background-image: linear-gradient(rgba(128, 0, 32, 0.738), rgb(111, 0, 255), salmon);
+    color: black;
+    text-shadow: 2px 2px rgba(0, 0, 255, 0.434);
+    border-bottom: 4px solid rgb(34, 34, 48);
+}
+
+.team-names {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    margin-top: 30px;
+}
+
+.team-score-label {
+    color: rgb(232, 241, 46);
+    text-shadow: 2px 2px rgba(0, 0, 255, 0.434);
+}
+
+.score {
+    border: 1px solid black;
+    margin: 30px 80px;
+    background-color: black;
+}
+
+.home-pt, .guest-pt {
+    font-family: "digital-7";
+    color: rgb(186, 35, 60);
+    font-size: 50px;
+}
+
+.pt-btn {
+    display: flex;
+    justify-content: center;
+    column-gap: 10px;
+}
+/*
+.add-pt-hbtn1, .add-pt-hbtn2, .add-pt-hbtn3, .add-pt-gbtn1, .add-pt-gbtn2, .add-pt-gbtn3 {
+    width: 50px;
+    background-color: rgb(16, 16, 39);
+    border-radius: 10px;
+    color: white;
+    padding: 5px;
+    font-size: 20px;
+    cursor: pointer;
+}*/
+
+.add-pt-btn {
+    width: 50px;
+    background-color: rgb(16, 16, 39);
+    border-radius: 10px;
+    color: white;
+    padding: 5px;
+    font-size: 20px;
+    cursor: pointer;
+}
+#start-btn {
+    justify-content: center;
+    margin: 5% 20% 5% 20%;
```

Comment:
- [ ] use `margin: (top and bottom value) (left and right value)` | `margin: 5% 20%`

Reply:
changed